presto-native-execution/presto_cpp/main/functions/README.md

```diff
@@ -0,0 +1,23 @@
+# Prestissimo: Dynamically Loading Function Library
+
+This library adds the ability to load User Defined Functions (UDFs) without having to fork and build Prestissimo through the use of shared libraries. The Prestissimo worker is to access said code. The dynamic functions are to be loaded upon running an instance of the presto server. In the presto server instance, it will search for any .so or .dylib files and load them using this library.
+
+## Getting started
+
+The Process can be roughly broken down into the steps below:
```

Comment:
```suggestion
```

Reply:
```suggestion
The process can be roughly broken down into the steps below:
```
I think you can delete this sentence entirely, but if you feel it valuable to keep, please lowercase "process".